tst/testinstall/methwhy.tst

```diff
@@ -91,124 +136,207 @@ function ( x, y... )
+#I  Method 3, applicable method number 1, value: 1
+#I  ``foobar: for an integer''
+#I   at stream:1
 function ( x )
     return 1;
 end
-gap> Display(ApplicableMethod(foobar, [1,2], 5));
+gap> ApplicableMethod(foobar, [1], 4, 2);
+#I  Searching Method for foobar with 1 arguments:
+#I  Total: 4 entries, of which 2 are applicable:
+#I  Method 4, applicable method number 2, value: 0
+#I  ``foobar''
+#I   at stream:1
+function( x ) ... end
+gap> ApplicableMethod(foobar, [1,2], 4);   
 #I  Searching Method for foobar with 2 arguments:
-#I  Total: 2 entries
-#I  Method 1: ``foobar: for two lists'' at stream:1, value: 17
+#I  Total: 2 entries, of which 1 is applicable:
+#I  Method 1, value: 17
+#I  ``foobar: for two lists''
+#I   at stream:1
 #I   - 1st argument needs [ "IsList" ]
 #I   - 2nd argument needs [ "IsList" ]
-#I  Method 2: ``foobar: for 2 integers'' at stream:2, value: 0
-function ( x, y... )
-    return 1;
-end
-gap> Display(ApplicableMethod(foobar, [1..3], 5));
+#I  Method 2, applicable method number 1, value: 0
+#I  ``foobar: for 2 integers''
+#I   at stream:2
+function( x, y... ) ... end
+gap> ApplicableMethod(foobar, [1..3], 4);
 #I  Searching Method for foobar with 3 arguments:
-#I  Total: 1 entries
-#I  Method 1: ``foobar: for 3 integers'' at stream:3, value: 0
-function ( x, y... )
-    return 1;
-end
-gap> Display(ApplicableMethod(foobar, [1..4], 5));
+#I  Total: 1 entries, of which 1 is applicable:
+#I  Method 1, applicable method number 1, value: 0
+#I  ``foobar: for 3 integers''
+#I   at stream:3
+function( x, y... ) ... end
+gap> ApplicableMethod(foobar, [1..4], 4);
 #I  Searching Method for foobar with 4 arguments:
-#I  Total: 1 entries
-#I  Method 1: ``foobar: for 4 integers'' at stream:4, value: 0
-function ( x, y... )
-    return 1;
-end
-gap> Display(ApplicableMethod(foobar, [1..5], 5));
+#I  Total: 1 entries, of which 1 is applicable:
+#I  Method 1, applicable method number 1, value: 0
+#I  ``foobar: for 4 integers''
+#I   at stream:4
+function( x, y... ) ... end
+gap> ApplicableMethod(foobar, [1..5], 2);
 #I  Searching Method for foobar with 5 arguments:
-#I  Total: 1 entries
-#I  Method 1: ``foobar: for 5 integers'' at stream:5, value: 0
-function ( x, y... )
-    return 1;
-end
-gap> Display(ApplicableMethod(foobar, [1..6], 5));
+#I  Total: 1 entries, of which 1 is applicable:
+#I  Method 1, applicable method number 1, value: 0
+#I  ``foobar: for 5 integers''
+#I   at stream:5
+function( x, y... ) ... end
+gap> ApplicableMethod(foobar, [1..6], 1);
 #I  Searching Method for foobar with 6 arguments:
-#I  Total: 1 entries
-#I  Method 1: ``foobar: for 6 integers'' at stream:6, value: 0
-function ( x, y... )
-    return 1;
+#I  Total: 1 entries, of which 1 is applicable:
+#I  Method 1, applicable method number 1, value: 0
+#I  ``foobar: for 6 integers''
+#I   at stream:6
+function( x, y... ) ... end
+gap> ApplicableMethod(foobar, [1..7], 1);
+#I  no method found for this operation with this number of arguments
```

Comment:
This test is failing (as can be seen in the Travis logs, or by running the `testinstall` locally), because it actually prints this error now:
```
Error, METHODS_OPERATION: <narg> must be an integer between 0 and 6 (not the integer 7)
```